### ISSUE TITLE
 Image binarization and exclusion zone

### ISSUE THREAD

**Issue by cakuba** [CRBS/cdeep3m#62]
Dear authors, 

We are trying to reproduce your result in a Docker and plan to use CDeep3M to MOST data as mentioned previously. It seems that most results are just fine, except that we got a little difference from Dice coefficient when predicting the labels for those 80 figures from sbem_mitochrondria_xy5.9nm40nmz using your 30K pre-trained model. We guess that there might be some techniques we might have missed in the image preprocessing as image binarization (we used Otsu) and the exclusion zone around the mitochondria border. 
      
So, I'm wondering if it is possible to kindly guide us how these image preprocessing in CDeep3M is implemented. Thanks again for your help.
Brett

**Comment by haberlmatt** [comment]
Hi Brett,

our calculations with the voxel exclusion zone for supplementary figure 3 was done to compare CDeep3M with the algorithm published by the Fua group a while ago.  (citation 13. Lucchi et al. Exploiting enclosing membranes and contextual cues for mitochondria segmentation)
For that we used their published training and validation data, meaning we trained Cdeep3m for 50000 iterations on their training data and calculated the dice coefficient on their validation data.
The data we used for this is available here https://cvlab.epfl.ch/data/data-em/

I suppose you mean postprocessing, to binarize the predictions before comparing to ground truth, we first ran parametersweeps on it to optimize binarization. I quickly checked, and compared to the otsu, which gives me often values of 120-128 on a 8 bit image, the performance as measured as F1 value or Jaccard index is better when thresholding around 185-190 depending on the image.

Hope this helps?
Matt

**Comment by cakuba** [comment]
Sure! Appreciate it. Will get back to you once we have more results. By the way, your model runs great on our own data!

Brett

**Comment by haberlmatt** [comment]
Ok, thank you for the feedback!
M

**Comment by cakuba** [comment]
Hi, Matt, 

      Just a quick follow-up as we implemented an exclusion zone as suggested. And yes, the results are better now (about 12%) and almost the same as reported in your great paper.  Thanks again for your patience. 

Brett
